### PR TITLE
vmm: openapi: fix integers format

### PR DESCRIPTION
**Pull request by devimc**
According to openAPI specification[1], the format for `integer` types
can be only `int32` or `int64`, unsigned integers are not supported.
This patch replaces `uint64` with `int64`.

[1]: https://swagger.io/specification/#data-types

Signed-off-by: Julio Montes <julio.montes@intel.com>

### CODE REVIEW

**Review by likebreath — APPROVED** [review]
Thanks for the PR. 

One nit: can you please use 'vmm: openapi:' as the prefix in the commit message. :)

**Comment by devimc** [comment]
@likebreath fixed, thanks

**Comment by sboeuf** [comment]
@devimc @likebreath there's still a few `uint64`, `uint32` and `uint8` remaining in the YAML file. Could you please take care of these?
BTW, I don't know if `uint8` is supported or not, I'm just pointing this out since this is yet another `uint`.

**Comment by likebreath** [comment]
@devimc Following our conversation earlier today, I confirmed it is not a problem for my concern about the inconsistency date type between the `yaml` file and the internal data structures.  For example, the `SgxEpcConfig.size` entry in the yaml is now `int64` while the we have `SgxEpcConfig::size` as `u64` in our code base. The  `serde_json` crate takes care of this issue and will report an error if we de-serialize from `signed` integer for `unsigned` elements (error will be reported [here](https://github.com/cloud-hypervisor/cloud-hypervisor/blob/master/vmm/src/api/http_endpoint.rs#L33)).

For the `uint8`, you can simply remove them, as `uint8` (or any format that is not `int32` or `int64`) will be ignored and will be treated as `int32` (the default for `integer` type) according to the OpenAPI spec [v3.0.1](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md#data-types). Again it is not a problem, as `serde_json` will make sure the de-serialized value is valid for `u8`, and will report error if it is not.  

In short, I believe it is safe to directly replace `uint` with `int`  (and remove `uint8`) in the `cloud-hyervisor.yaml` without any additional changes in the code base. Please go ahead and send a PR to fix the `uint` as @sboeuf advised earlier. Thanks a lot.
